README.md

```diff
@@ -94,12 +94,21 @@ git checkout -b your-feature-branch-name
 ```bash
 git add .
 git commit -m "Add your commit message here"
+``` 
+3. Running Test
+```bash
+  bundle exec rspec
+```
+
+4. Cleaning up before pushing
+```bash
+  bundle exec rubocop -A
```

Comment:
I don't think it's a great idea to recommend running rubocop with the `-A` flag. This is because `-A` will run autocorrection over "unsafe cops" -- that is lint offenses where the autocorrection may change how the code functions. I'd probably prefer this to say `bundle exec rubocop -a` and then the developer can manually clean up any remaining offenses.

Reply:
Will do! Thanks for the feedback